### PR TITLE
Ensure active policies have at least one active scanner

### PR DESCRIPTION
**Pull request by jshoberg**

### CODE REVIEW

File: dash/backend/src/modules/policy/services/policy-service.ts [hunk 1]
```diff
@@ -93,5 +94,15 @@ export class PolicyService {
         return await this.auditLogService.calculateMetaData(previous, updated, 'Policy');
     }
 
+    validateActivePolicyScanners(policy: PolicyDto, scanners: ScannerDto[]): boolean {
+        // Checks if an active policy has at least one active scanner
+        if (policy.enabled) {
```
Comment (jacobbeasley) [inline dash/backend/src/modules/policy/services/policy-service.ts:99]: Might be able to simplify using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Comment (jacobbeasley) [reply dash/backend/src/modules/policy/services/policy-service.ts:99]: return !policy.enabled || scanners.some(...)

Comment (jacobbeasley) [reply dash/backend/src/modules/policy/services/policy-service.ts]: return !policy.enabled || scanners.some(scanner?.enabled);

Comment (jshoberg) [reply dash/backend/src/modules/policy/services/policy-service.ts]: Haven't run into that function before, thanks

**Pull request merged**